### DISCUSSION TITLE
Contact Form

### DISCUSSION THREAD

**Discussion by malekdev97** [arifszn/ezfolio#11] [Q&A]
Helle everyone, and Thank you for creating this awesome dynamic portfolio. 

My question is form contact works? if yes then how? 
I have tried to set SMTP and test it. the show notification!

I'm happy to work on the form and share it with everyone.

**Comment by arifszn** [comment]
Hello, thanks for taking interest in this project. 

Currently, when a visitor submits the contact form, a db entry is inserted only. No real email is sent to the user. And there is no feature for replying through the app. You have to manually reply using your mail.

I am happy to accept any PR which may improve the form. Thanks.
